### PR TITLE
Get @BeforeEach annotations woring

### PR DESCRIPTION
**Pull request by connernilsen**
We should only need to import the `<artifactId>junit-jupiter</artifactId>` dependency for JUnit (I thought it had to be the `-api` version, so I converted dependencies to that on the last PR, sorry 😞)

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -92,7 +92,10 @@
           </execution>
         </executions>
       </plugin>
+      </plugins>
+    </pluginManagement>
 
+    <plugins>
```
Comment (connernilsen) [inline pom.xml:98]: I'm not sure if this should be moved down to here (from above), but this is how it is set up in LLB.

**Review by liammoyn — COMMENTED** [review]
Why are we moving coveralls and jacoco out of plugins and into pluginManagement? That seems like a potentially breaking change and even if it's not it's unrelated enough that it should be done in a separate PR

**Comment by connernilsen** [comment]
@liammoyn I moved it out of plugins and into plugin management because that's how it is in LLB. I'm not sure which one is correct, so I'm bringing it up here to ask which one is right. It could be moved to a separate PR though

File: service/src/test/java/com.codeforcommunity/processor/ProcessorImplTest.java [hunk 2]
```diff
@@ -44,4 +44,8 @@ public void testFetchExistsFalse() {
     assertFalse(db.fetchExists(db.selectFrom(Tables.USERS).where(Tables.USERS.ID.eq(1))));
     assertEquals(1, mockDb.getSqlOperationBindings().get(OperationType.EXISTS).size());
   }
+
```
Comment (liammoyn) [inline service/src/test/java/com.codeforcommunity/processor/ProcessorImplTest.java:47]: Doesn't this need an `@Test`?

**Review by liammoyn — COMMENTED** [review]
I think we need to be confident either way before we merge this. @jackblanc did you set up coveralls / do you know if it should be in plugins or pluginManagement?

**Review by jackblanc — APPROVED** [review]
Looks like coveralls changes were removed? This seems reasonable? I don't even know what a pom is

File: pom.xml [hunk 3]
```diff
@@ -41,26 +40,30 @@
...
-          </executions>
-          <configuration>
-            <mainClass>com.codeforcommunity.ServiceMain</mainClass>
-          </configuration>
-        </plugin>
       </plugins>
     </pluginManagement>
+
     <plugins>
+      <!-- Used for running the main class -->
```
Comment (connernilsen) [inline pom.xml:47]: can delete

**Review by liammoyn — APPROVED** [review]

**Pull request merged**